plugins/Marketplace/vue/src/OverviewIntro/OverviewIntro.vue

```diff
@@ -88,7 +86,6 @@ export default defineComponent({
     isPluginsAdminEnabled: Boolean,
     isMultiServerEnvironment: Boolean,
     hasSomeAdminAccess: Boolean,
-    hasLicenseKey: Boolean,
```

Comment:
I think that one is still defined in the Twig template and the Controller. Should be safe to remove for the overview?

Reply:
good catch, removed from both places